### PR TITLE
Task 66 standalone listing backend

### PR DESCRIPTION
**Pull request by Rebooting-Me**
Created new user routes `/:homeowner_id/listings` for only Homeowners trying to view their listing(s) and `/listings/:listing_id` for both HO and CO trying to view a stand-alone listing. The `/:homeowner_id/listings` route will use `/api/homeowner` route and the `/listings/:listing_id` route will use the `/api/user` route since this is stand-alone listing. This more or less culminates the Homeowner's side Sprint 2 tasks.

### CODE REVIEW

File: backend/controllers/getListingController.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+const Listing = require('../models/listingModel');
+const Homeowner = require('../models/homeownerModel');
+
+// function to view stand-alone listing [common to HO and CO]
+const getListing= async (req, res) => {
+    try {
+      // Find the listing by ID
+      const listing = await Listing.findById(req.params.listing_id).select('-homeowner_id');
```
Comment (and-le) [inline backend/controllers/getListingController.js:8]: Is the hyphen required here?

Comment (Rebooting-Me) [reply backend/controllers/getListingController.js:8]: Yes. This is excluding the homeowner_id to be returned in the response body to the user.

**Review by and-le — CHANGES_REQUESTED** [review]
Please see the comment for getListing()

File: backend/controllers/getListingController.js (hunk 1 above)
Comment (and-le) [reply backend/controllers/getListingController.js:8]: I see, this makes sense.

**Review by and-le — APPROVED** [review]

**Pull request merged**